en/components/grids_templates/selection.md

```diff
@@ -184,6 +184,60 @@ The template we are going to use to combine the grid with the context menu:
     </div>
 </div>
 ```
+
+```ts
+public rightClick(eventArgs: any) {
```

Comment:
This seems duplicated, isnt it?

![selection](https://user-images.githubusercontent.com/14248932/207870379-653c86a2-6248-4317-babe-062675eb67c1.gif)